web/api/badges/web_buffer_svg.c

```diff
@@ -838,7 +721,7 @@ void buffer_svg(BUFFER *wb, const char *label, calculated_number value, const ch
     value_width = verdana11_width(value_string) + (BADGE_HORIZONTAL_PADDING * 2);
     total_width = label_width + value_width;
 
-    escape_xmlz(label_escaped, label_buffer, LABEL_STRING_SIZE);
+    escape_xmlz(label_escaped, label, LABEL_STRING_SIZE);
```

Comment:
currently verdana11_width is just used to guess the size and should IMHO not remove any characters

Reply:
You missed the question. I don't have a problem with not removing characters, but you made a decision and left behind code that became useless after that decision. The str copy and everything related to `char* d` inside the function should be removed, since there's no reason any more to update the string.

Reply:
OK thanks for clarification. I see your point and you are right it can be "cleaned up".

Reply:
updated in 80860f2
although as mentioned i suggest reworking this function to be able to properly handle unicode chars in future
i can eventually do this

Reply:
feb5711 and 35ea7ea solve this